docs/cookbook/using-docker.md

```diff
@@ -154,9 +154,9 @@ services:
 
 The spacing in this file is very important. Whitespace, not tab, indentation indicates that a particular line is nested within the object passed on the line above it. Walking through this file, it starts with `services:`. From the indentation, we can see that we are creating two services - a `db:` container and a `web:` container.  Much like our `Dockerfile`, within the `db:` we start by specifying an image to run. In this case, it is the `mongo:4.4.14` official image for running MongoDB v4.4.14. Other images can be found in the docker library GitHub repo [README](https://github.com/docker-library/docs/blob/master/mongo/README.md#supported-tags-and-respective-dockerfile-links). You should use the version that mirrors your development environment.
 
-Next, we are specifying that the database should communicate over port `27018`. This is different from the port typically used in order to direct communication to the dockerized version and not a local MongoDb. If you need your database to communicate over a different port, you have to change it here and in your `.env` file.
+Next, we are specifying that the database should communicate over port `27018`. This is different from the port typically used in order to direct communication to the dockerized version and not a local MongoDB. If you need your database to communicate over a different port, you have to change it here and in your `.env` file.
 
-Finally, we add a volume for the MongoDB storage engine to write files into. You shouldn't need to change this.
+Finally, we add a volume for the MongoDB storage engine to write files into. You shouldn't need to change this. Without the persistent volume at this stage, w the database would appear to work, but all content would be lost on every restart.
```

Comment:
lose the "w " before "the database"

I think you had "while" and changed your mind to "but" which is fine but there's still a stray two characters

Reply:
Changed.